src/main/webapp/partials/instructorHelpQuestions.jsp

```diff
@@ -358,589 +358,1614 @@
-                      <div class="row">
-                        <div class="col-sm-4 text-color-gray">
-                          <strong>
-                            Response Summary
-                          </strong>
-                        </div>
-                      </div>
-                      <div class="row">
-                        <div class="col-sm-4">
-                          <table class="table margin-0">
-                            <thead>
-                            <tr>
-                              <td>
-                                Choice
-                              </td>
-                              <td>
-                                Response Count
-                              </td>
-                              <td>
-                                Percentage
-                              </td>
-                            </tr>
-                            </thead>
-                            <tbody>
-                            <tr>
-                              <td>
-                                Team 1
-                              </td>
-                              <td>
-                                1
-                              </td>
-                              <td>
-                                50%
-                              </td>
-                            </tr>
-                            <tr>
-                              <td>
-                                Team 2
-                              </td>
-                              <td>
-                                1
-                              </td>
-                              <td>
-                                50%
-                              </td>
-                            </tr>
-                            </tbody>
-                          </table>
-                        </div>
+              <div class="panel-heading" data-target="#panelBodyCollapse-1" id="panelHeading-1" style="cursor: pointer;">
+                <form style="display:none;" id="seeMore-1" class="seeMoreForm-1">
+                  </form>
+                  <div class="display-icon pull-right"><span class="glyphicon pull-right glyphicon-chevron-up"></span></div>
+                <form method="post" class="inline">
+                      <div id="DownloadQuestion-1" class="inline">
+                        <input id="button_download-1" type="submit" class="btn-link text-bold padding-0 color-inherit" data-toggle="tooltip" title="" name="fruploaddownloadbtn" value="Question 1:" data-original-title="Download Question Results">
                       </div>
+                    </form>
+                    <div class="inline panel-heading-text">
+                      <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
+                      <span class="text-preserve-space">Did you understand today's lecture?&nbsp;<span style=" white-space: normal;">
```

Comment:
I don't think this is a good question to demo the weight feature.

It doesn't make sense to talk about per recipient statistics for self-feedback question right?

Reply:
You are right. I should have noticed it. Changing it now.